blog-api/src/Auth/AuthRequest.php

```diff
@@ -17,25 +18,31 @@
  *      @OA\Property(example="Opal1144", property="password", format="string"),
  * )
  */
-final class AuthRequest extends RequestModel implements RulesProviderInterface
```

Comment:
Do we still need request model package then in `composer.json`?

Reply:
Not need. Removed request model dependency.